public/examples/integrations/toolkits/github/count_stargazers_example_call_tool.py

```diff
@@ -18,7 +18,7 @@
 
 tool_input = {
     "owner": "ArcadeAI",
-    "name": "hello-world",
+    "name": "arcade-ai",
```

Comment:
for these github examples, I made https://github.com/ArcadeAI/example for things that write (besides star the repo).

Otherwise I use the https://github.com/ArcadeAI/arcade-ai repo.

Reply:
This is a great idea